packages/dom/src/tabbable.js

```diff
@@ -166,7 +166,7 @@ export function findPrevious( element ) {
 	const index = focusables.indexOf( element );
 
 	// Remove all focusables after and including `element`.
-	focusables.length = index;
+	focusables.length = index !== -1 ? index : 0;
```

Comment:
If length is `0`, that would make it an empty array. So would that make the function return `undefined`?

If that is the case, I think it'd be good to add an early return from the function for clarity—that way anyone reading doesn't have to try to reason about the empty array and what happens in `filterTabbable`, they can bypass that code.

I notice the doc block is missing an `@return`, it'd be great to add that while we're here.

Reply:
Done in [d1a1980](https://github.com/WordPress/gutenberg/pull/38961/commits/d1a19806776b3b9226375e52d3ce917f16eefde7).